src/borda_clj/client.clj

```diff
@@ -16,30 +17,52 @@
         (do (print "borda buffer full, discarding measurement" dimensions values)
             measurements))))) ; buffer full
 
+(defn go-reduce-reports [in-ch out-ch kill-ch max-buffer-size]
```

Comment:
Maybe name `in-ch` and `out-ch` to `report-ch` and `measurements-ch` for consistency?